apps/core/scenes/search/Search.js

```diff
@@ -150,27 +43,9 @@ class Search extends React.Component<Props> {
           <View style={styles.illustrationWrapper}>
             <Illustration name="Boarding" style={styles.boardingIllustration} />
           </View>
-          <SearchFormModes />
-
-          <View style={desktopLayout && styles.inputsDesktop}>
-            <Placepickers />
-            <Datepickers />
-
-            <View
-              style={[styles.bottom, desktopLayout && styles.bottomDesktop]}
-            >
-              <Button
-                onPress={this.handleSubmitPress}
-                label="Search"
-                rightIcon={
-                  Platform.OS === 'web' ? <Icon name="chevron-right" /> : null
-                }
-              />
-            </View>
-          </View>
-          <View style={styles.bottomSpacer} />
+          <SearchForm onSubmit={this.props.onSubmit} />
+          {Platform.OS !== 'web' && <View style={styles.bottomSpacer} />}
```

Comment:
Last thing, otherwise looks good. `bottomSpacer` actually have some web related style. It's not that important and simply just as I've mentioned prevents inputs to stick to the bottom edge on really small screens.
The option is to remove this condition or that (in this case unused) style starting on row 96.